### PR TITLE
Address TianoCore review comments for "Platform/Hummingboard: Add i.MX 6Quad Hummingboard Edge ACPI tables" Patch

### PR DESCRIPTION
**Pull request by christopherco**
TianoCore upstream review comments: https://lists.01.org/pipermail/edk2-devel/2018-December/034059.html

Fixes for comments were applied to all platforms and not just Hummingboard Quad.

### CODE REVIEW

**Review by jordanrh1 — APPROVED** [review]
Someone related but if you get a chance it would be nice to deduplicate `IMX_FAMILY` and `IMX_CHIP_TYPE`.

**Review by neilsh-msft — APPROVED** [review]

File: Silicon/NXP/iMX6Pkg/AcpiTables/Csrt.aslc [hunk 1]
```diff
@@ -31,10 +31,10 @@ typedef enum {
   GPT_CLOCK_HI_FREQ     = 2,
   GPT_CLOCK_EXT         = 3,
   GPT_CLOCK_LOW_FREQ    = 4,
-#if defined(CPU_IMX6DQ) || defined (CPU_IMX6DQP)
+#if defined(CPU_IMX6D) || defined(CPU_IMX6Q) || defined(CPU_IMX6DP) || defined(CPU_IMX6QP)
   GPT_CLOCK_OSC_DIV_8   = 5,
   GPT_CLOCK_OSC         = 7,
-#elif (defined(CPU_IMX6SDL) || defined(CPU_IMX6SX)) || defined(CPU_IMX6ULL)
+#elif (defined(CPU_IMX6S) || defined(CPU_IMX6DL) || defined(CPU_IMX6SX)) || defined(CPU_IMX6ULL)
```
Comment (christopherco) [inline Silicon/NXP/iMX6Pkg/AcpiTables/Csrt.aslc]: Fix parentheses

File: Silicon/NXP/iMX6Pkg/Drivers/GopDxe/Display.c [hunk 2]
```diff
@@ -147,7 +147,7 @@ InitDisplay (
     Status = EFI_OUT_OF_RESOURCES;
     goto Exit;
   }
-#if !defined(CPU_IMX6SDL)
+#if !defined(CPU_IMX6S) || defined(CPU_IMX6DL)
```
Comment (christopherco) [inline Silicon/NXP/iMX6Pkg/Drivers/GopDxe/Display.c]: Fix this logic. It is not a correct translation

File: Silicon/NXP/iMX6Pkg/Drivers/GopDxe/Display.h [hunk 3]
```diff
@@ -24,7 +24,7 @@ typedef enum {
 
 typedef enum {
   IPU1,
-#if !defined(CPU_IMX6SDL)
+#if !defined(CPU_IMX6S) || defined(CPU_IMX6DL)
```
Comment (christopherco) [inline Silicon/NXP/iMX6Pkg/Drivers/GopDxe/Display.h]: Fix this logic. It is not a correct translation

File: Silicon/NXP/iMX6Pkg/Drivers/PlatformSmbiosDxe/PlatformSmbiosDxe.c [hunk 4]
```diff
@@ -1524,10 +1524,10 @@ MemoryDeviceInfoUpdateSmbiosType17 (
 
   // 08h - Total Width
   // 0Ah - Data Width
-#if defined(CPU_IMX6DQ) || defined(CPU_IMX6DQP) || defined(SOCTYPE_DUALLITE)
+#if defined(CPU_IMX6D) || defined(CPU_IMX6Q) || defined(CPU_IMX6DP) || defined(CPU_IMX6QP) || defined(CPU_IMX6DL)
```
Comment (christopherco) [inline Silicon/NXP/iMX6Pkg/Drivers/PlatformSmbiosDxe/PlatformSmbiosDxe.c]: nit - 80 char limit

**Pull request merged**